website/static/css/custom.css

```diff
@@ -157,16 +174,18 @@ img.logo {
 }
 
 /* fonts and headers */
-h1 {
-    font-size: 3em;
+h1, h2, h3, h4, h5, h6 {
+    font-family: BwSeidoRound-Medium,sans-serif;
 }
 
-h1,h2 {
-    font-family: BwSeidoRound-Bold;
+h1 {
+    font-size: 2em;
```

Comment:
If possible, I recommend we keep it consistent between em, rem and px. My pov is always to go with px because of the ease of use and to avoid relative size. There are arguments for using em/rem to scale between devices better but overall I still think px works best. In addition, creating variables for font sizes would be useful to keep track of the different sizes used, and to help with consistency.

Reply:
added a variables for all font sizes and colors.  converted to px.

Reply:
✅